assets/index.css

```diff
@@ -1,154 +1,130 @@
-*{
-    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
-    text-decoration-color: rgba(168, 46, 46, 0.678);
+* {
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  text-decoration-color: rgba(168, 46, 46, 0.678);
 }
 
-
 body {
-    background-color: black;
-    margin: 0;
-
+  background-color: black;
+  margin: 0;
 }
 
-
 nav {
-    background-color: rgb(63, 61, 61);
-    padding: 0 15px;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    box-shadow: wheat;
+  background-color: rgb(63, 61, 61);
+  padding: 0 15px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: wheat;
 }
 .menu {
-    color: white;
-    list-style-type: none;
-    display: flex;
-    align-items: center;
+  color: white;
+  list-style-type: none;
+  display: flex;
+  align-items: center;
 }
 .menu li {
-padding: 10px;
-font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  padding: 10px;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
 }
 .links {
-     text-decoration: none;
-     color: white;
-
+  text-decoration: none;
+  color: white;
 }
-#foot{
-    font-size: 1em;
-    color: white;
-    font-weight: bold;
-    margin-bottom: -.3em;
+#foot {
+  font-size: 1em;
+  color: white;
+  font-weight: bold;
+  margin-bottom: -0.3em;
 }
 .centerdiv {
-    display: grid;
-    padding: 0.5em;
-    padding-left: 1em;
-    grid-template-columns: 70px 70px 70px;
-    
-}
-main{
-    display: grid;
-    grid-template-columns: 65% 35%;
-    grid-template-rows: 1fr 1fr 1fr;
-
+  display: grid;
+  padding: 0.5em;
+  padding-left: 1em;
+  grid-template-columns: 70px 70px 70px;
 }
-.box1 {
-    background: rgba(54, 54, 54, 0.178);
-    height: 9em;
-    
+main {
+  display: grid;
+  grid-template-columns: 65% 35%;
 }
-.box2 {

```

Comment:
both your title and your bio live now on your box1, so you don't need two different clases